v1/main.go

```diff
@@ -11,7 +11,7 @@ func main() {
 		userAgent := r.Header.Get("User-Agent")
 		log.Printf("%s - [%s %s %s] 200 [%s]", r.RemoteAddr, r.Method, r.URL.Path, r.Proto, userAgent)
 		w.Header().Set("Server", "Hello/v1")
-		fmt.Fprintf(w, "Hello, World!\n")
+		fmt.Fprintf(w, "Hello, World!\nVersion: 1")
```

Comment:
Rather than have 2 main.go files with the same content, why not  to receive the version as an argument?

Ex.
```
version := os.Args[1]
...
w.Header().Set("Server", fmt.Sprintf("Hello/v%s", version))

fmt.Fprintf(w, fmt.Sprintf("Hello, World!\nVersion: %s", version))
...
```

Reply:
That's a fair point, I'm not sure why they were separated originally - you would need to check with @thiagoavadore to be sure.Though made the assumption it was to more clearly show in a demo scenario distinction between v1 and V2.

Feel free to make the change, but I believe the important part was two separate images so in a demo we can switch seamlessly between them on the pods. In comparison to 'the old/bad' way of delivering which would show down time.

Reply:
In a PRD code this would make a lot of sense, but just for the sake of explanation during the demo I thought it would be better to have 2 different files. But you guys are the ones leading the demo... feel free to do how you want to.

Reply:
Ok, I'm going to merge this PR, later we can fix this "issues."